en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,191 @@
+
+### Silverstripe CMS major release
+
+A new Silverstripe CMS major version series (e.g. "CMS 6") is introduced by its first Silverstripe CMS release (e.g. "CMS 6.0").
+
+The main purpose of a major release is to deliver significant architectural updates and changes that are not backward compatible. This ensures the Silverstripe CMS major version series can be supported for its full lifecycle, including updates to third-party dependencies.
+
+At launch, new Silverstripe CMS major versions may contain new features not present in the previous major version series. However, shipping new features is a secondary concern, as new features will usually ship in Silverstripe CMS minor releases.
+
+- **Backward compatibility:** No - projects will require some development work to upgrade to a new major release
+- **Cadence:** Every two years, between April and June
+- **Pre-release schedule:**
+  - **Alpha**: At the discretion of the development team
+  - **Beta**: At least 3 months prior to the anticipated Silverstripe CMS major release. At this point, dependencies should not change
+  - **RC**: Approximately 4 weeks prior to the anticipated Silverstripe CMS major release. At this point, no more breaking changes can be made unless they are to fix a regression
+- **Documentation:** A changelog is published listing significant new features and breaking changes. A [clear upgrade path](/upgrading) is available
+
+### Silverstripe CMS minor release
+
+A Silverstripe CMS minor release is the standard recurring release of Silverstripe CMS (e.g. "CMS 6.1", "CMS 6.2"). Minor release upgrades are generally more straightforward to manage than major release upgrades.
+
+- **Purpose:** Ship new features and enhancements
+- **Backward compatibility:** Yes - within their Silverstripe CMS major version series
+- **Cadence:** Every six months within an active Silverstripe CMS major version series. Minor releases are targeted for April and October each year
+- **Pre-release schedule:**
+  - **Alpha**: Typically not released
+  - **Beta**: Approximately 6 weeks prior to the anticipated Silverstripe CMS minor release. At this point, no new features or new public API should be added
+  - **RC**: Approximately 2 weeks prior to the Silverstripe CMS release. At this point, only bug fixes should be added
+- **Documentation:** A changelog is published listing significant new features
+
+> [!NOTE]
+>
+> While each Silverstripe CMS release usually only gets one of each type of pre-stable release, additional pre-stable releases may be tagged if required.
+>
+> Patch releases for individual modules such as `silverstripe/framework` are not considered a "Silverstripe CMS release", however they are an important part of meeting our support commitments.
+
+## Support lifecycle
+
+> [!INFO]
+> For a detailed timeline of current and planned support durations for each Silverstripe CMS version, refer to the [support timeline](https://www.silverstripe.org/software/roadmap).
```

Comment:
```suggestion
> For a detailed timeline of current and planned support durations for each Silverstripe CMS version, refer to the [support timeline](https://www.silverstripe.org/software/roadmap#support-timeline).
```
I _assume_ that will be the anchor once the page has been updated.

Reply:
Yup